loading.bs

```diff
@@ -611,9 +612,9 @@ returns the result of the following steps:
     : [=request/mode=]
     :: "`cors`"
 1. Let |certResponse| be the result of [=fetching=] |certRequest|.
-1. If |certResponse|'s [=response/status=] is not `200`, or the [=supplied MIME
-    type=] of |certResponse| is not `"application/cert-chain+cbor"`, return a
-    failure.
+1. If |certResponse|'s [=response/status=] is not `200`, or [=header
+    list/extracting a MIME type=] from |certResponse|'s [=response/header list=]
+    does not return `"application/cert-chain+cbor"`, return a failure.
```

Comment:
I suspect you want to return the essence of the returned MIME type, meaning this will have to be slightly more involved. (It was a little ambiguous already. E.g., do parameters matter?)

Reply:
I intended to forbid parameters here, since https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#internet-media-type-applicationcert-chaincbor doesn't define any. I think the current text accomplishes that? If you think it's better to ignore parameters (by using the essence), I'm happy to do that instead.